recipe/meta.yaml

```diff
@@ -26,7 +28,7 @@ requirements:
     - docker-pycreds  >=0.3.0
     - ssl_match_hostname >=3.5  # [py<35]
     - ipaddress >=1.0.18  # [py<33]
-    - pypiwin32 >=219  # [win32]
+    - pywin32 223  # [win and py>=36]
```

Comment:
We need a `223` version for 3.8 first. Let me see if I can make that happen.

Reply:
I could not make that version work, see https://ci.appveyor.com/project/conda-forge/pywin32-feedstock/builds/28829229/job/5mjdmgfsjwvfh89u

If you know how to fix that please send a PR to the v223 branch.

Reply:
I haven't tested but I guessed this commit might help: https://github.com/mhammond/pywin32/commit/1a49ada4e2ff5db7543d6f9dafe55b74fcfb8f06

However, I would prefer to skip the `Windows Python 3.8` build for now and we should try to rework this https://github.com/docker/docker-py/pull/2047 and get it migrated to a new `pywin32` from the upstream.